### DISCUSSION TITLE
(New Connector Profile) AWS Batch

### DISCUSSION THREAD

**Discussion by gillins** [aws/serverless-application-model#3607] [Ideas]
Bring able to submit jobs to AWS Batch is a handy thing for Lambda to do. Would it be possible for this to be added as a Connector so the permissions are all set up for this?

The JobDefinition and JobQueue would need to be specified I think.

**Comment by GavinZZ** [comment]
Going to turn this feature request into a discussion. This will not be prioritized at the moment, however, if it receives enough 👍 we will re-prioritize it.
